Source/OnlineSubsystemEpic/Private/OnlineSubsystemEpicTypes.h

```diff
@@ -7,50 +7,17 @@
 
 TEMP_UNIQUENETIDSTRING_SUBCLASS(FUniqueNetIdEpic, EPIC_SUBSYSTEM);
 
-namespace ELoginType
+UENUM()
+enum class ELoginType : uint8
```

Comment:
the epic docs recommend using this format, it makes the enums blueprint friendly and allows for reflection to convert the enums to strings, see the template methods added in util.cpp